resources/views/blocks/tamara-widget.php

```diff
@@ -0,0 +1 @@
+<tamara-widget type="tamara-summary" inline-type="<?php echo esc_attr( $widget_inline_type ); ?>" amount="{{ $widget_amount }}"></tamara-widget>
```

Comment:
You may want to check this @loikimle 
- We can have 2 ways to escape attributes value, `<?php echo esc_attr( $widget_inline_type ); ?>` is the WordPress way, `{{ $widget_amount }}` the the blade template way